apps/explorer/src/components/Activity/index.tsx

```diff
@@ -54,7 +57,7 @@ export function Activity({
     return (
         <div>
             <TabGroup
-                size="lg"
+                size={isMediumOrAbove ? 'lg' : 'md'}
```

Comment:
We should use this sparingly, ideally we can just make these responsive. Is this something we consistently want to do?

Reply:
@mystie711 Can you comment on this? Do we want us to be able to toggle the sizes for the Tab group, or should we just make them responsive to not relying on checking for sizes.

Reply:
Yes. For each card in Explorer would like to do responsive designs.
That is the main intent @Jordan-Mysten

Reply:
I realize there are bigger lifts required to make everything responsive. Plan to tackle this topic in a more comprehensive way in future design delivery.

Reply:
![Screenshot 2023-04-10 at 6 08 58 PM](https://user-images.githubusercontent.com/127577476/231029284-4f760a66-ce28-4799-8de2-66777817c3ca.png)

@mystie711 This refers to the tabs in this area here. I've noticed the size differences between desktop and mobile viewport. So just want to confirm.

Reply:
reverted this change for now